### PR TITLE
Fix hamburger menu on login/create user and add webpage title and icon

### PR DESCRIPTION
**Pull request by J-Dees**
# Summary of changes
- I removed the hamburger menu from the login and create user screen. The menu was allowing a user to bypass login and navigate straight to any of the pages in the menu
- I changed the webpage title from the boring "Vite + React" to a glorious title, "Nibble and Scribble". I also replaced the default Vite icon with an AI generated journal with sticky note colors. (@Mcray3000 I am hoping that you can create a better image with whatever software you used for the entry icons)
- The create user menu now sends the user back to login. Once successfully logged in, the user is provided with an auth token.

### Additional Notes
This PR contains code from #70 and #71 (they have both been approved but are awaiting feedback). We should have full functionality in our local testing environments and hopefully in deployment.

### CODE REVIEW

**Review by TLMatthies — APPROVED** [review]
This is great, @Mcray3000 this has all the changed you made in https://github.com/Mcray3000/307_MyDiary_Nibble/pull/71 . It might be nice to close that pr and just approve this one. This also has the code from my https://github.com/Mcray3000/307_MyDiary_Nibble/pull/70 , so that one could be closed too.

**Review by eamyzzaj — APPROVED** [review]
These changes look good. The title and icon are awesome additions, good idea!

**Comment by Alaxxn** [comment]
Awesome, that Icon is so much better! We should get this merged ASAP it has a lot of the key components that make our app run.

**Review by Alaxxn — APPROVED** [review]

**Comment by Mcray3000** [comment]
I love it, It's a nice button; I'll come up with a similar one later. 
My only notes would be: 

- I would prefer the menu icon to remain static on the screen so it doesn't move when you go to the create entry page.
- Also, in the future, we should be able to access the public page without login auth. That way someone could see what the site is all about before joining :)  

**Looks good to me!**

File: packages/react-frontend/public/nibbleandscribble.png
Comment (Mcray3000) [inline packages/react-frontend/public/nibbleandscribble.png:1]: We will also make this an SVG so it scales properly for web apps and matches the rest of the icons

File: packages/react-frontend/src/DiaryEntry.jsx [hunk 1]
```diff
@@ -105,6 +106,7 @@ function DiaryEntry() {
 
   return (
     <div className="diary-container">
+      <HamburgerMenu />
```
Comment (Mcray3000) [inline packages/react-frontend/src/DiaryEntry.jsx:109]: To change

**Review by Mcray3000 — CHANGES_REQUESTED** [review]
Just address the hamburger menu not being where intended then you're good to merge :)

File: packages/react-frontend/src/DiaryEntry.jsx (hunk 1 above)
Comment (J-Dees) [reply packages/react-frontend/src/DiaryEntry.jsx:109]: Thanks for pointing this out. I didn't even check all the different pages to make sure it stayed in the top corner. I will fix this issue now.

File: packages/react-frontend/public/nibbleandscribble.png
Comment (J-Dees) [reply packages/react-frontend/public/nibbleandscribble.png:1]: That is awesome. I really like the buttons you made for the diary entries, so I trust you can make a better icon than the one I created on the fly.

**Review by Mcray3000 — APPROVED** [review]
Looks good to me!

**Pull request merged**